packages/components/multiselect/src/Multiselect.tsx

```diff
@@ -136,7 +142,7 @@ const countMatchingChildren = (
 
 function _Multiselect(props: MultiselectProps, ref: React.Ref<HTMLDivElement>) {
   const {
-    className,
+    className = '',
```

Comment:
Do we really need it?

Reply:
oh that was just an experiment i forgot to remove again. thanks